pkg/telemetryquerylanguage/tql/condition_test.go

```diff
@@ -94,10 +94,28 @@ func Test_newConditionEvaluator(t *testing.T) {
 			name: "no condition",
 			cond: nil,
 		},
+		{
+			name: "compare Enum to int",
```

Comment:
We could use a couple more tests in this context:
* Enum on the right
* Enum not found

Reply:
Enum not found is an interesting scenario. I'm away from my computer right now but if I remember my implementation correctly if the Path doesn't match any Enums it'll then be parsed as a Path, which will fail. This would be caught during config validation. 

I can add a test scenario to verify this.